### DISCUSSION TITLE
Thorium is slower (compared to Chrome) on Linux

### DISCUSSION THREAD

**Discussion by ThePython10110** [Alex313031/thorium#709] [Q&A]
On Windows 10, Thorium was definitely faster than Google Chrome. Since switching to Linux, however, Thorium seems to be slightly worse. Any idea why?
```
OS: 64-bit Linux Mint
Window system: X11
Window manager: DWM
CPU: Intel Core i7-6700
GPU: Intel HD Graphics 530
RAM: 8GB
```
Tested with the latest version of Google Chrome and Thorium, with empty profiles, using [BrowserBench](https://browserbench.org).

Chrome was *slightly* faster on Speedometer 3.0, about the same with Jetstream, and significantly better with MotionMark (680-something compared to Thorium's 564.94).

**Comment by gz83** [comment]
Close discussions that are outdated and inactive.
